72.md

```diff
@@ -55,6 +55,50 @@ Any Nostr event can be a post request. Clients MUST add the community's `a` tag
   "content": "<My content>"
 }
 ```
+# New Post Request (Exclusive to community audience)
+
+When we use kind 1, the note can show up in other clients without context, creating confusion.
+
+To create a post exclusive to a community's audience, clients MUST use kind `4549` with the same `a` tag as above and MAY add an additional `k` tag with value `"1"`
+
+```json
+{
+  "id": "<32-bytes lowercase hex-encoded SHA-256 of the the serialized event data>",
+  "pubkey": "<32-bytes lowercase hex-encoded public key of the event creator>",
+  "created_at": <Unix timestamp in seconds>,
+  "kind": 4549,
+  "tags": [
+    ["a", "34550:<Community event author pubkey>:<d-identifier of the community>", "<Optional relay url>"],
+    ["k", "1<Optional>"]
+  ],
+  "content": "<My content>"
+}
+```
+
+# Support for Different Event Kinds
+Nostr communities are not just for kind `1` events.  Clients can send any valid nostr event exclusively to a specific community's audience by wrapping the event in a `4549` kind event.
+
+While sending a non kind `1` event, the clients MUST populate the `content` field of the event with the _the stringified JSON of the event_.  This is similar to the ones used for reposts in [NIP-18](18.md).
+
+Additionally, for non kind `1` events, the clients MUST add a `k` tag with value of the kind of event to be posted.
```

Comment:
```suggestion
Additionally, the clients MUST add a `k` tag with value of the kind of event to be posted.
```

The `k` tag should be a requirement. Because if my community client only supports kind 1 and kind 30023, I can fetch with the filters `kinds: [1, 30023], #a: ...` and `kinds: [4549], #k: ["1", "30023"], #a: ...` just the post approval kinds it supports.

If people don't fill the `k` tag when its a kind-1-like event, the last filter will fail.

It is the same reason why "Post Approval by moderators" must have a "k" tag.